learning_resources/models.py

```diff
@@ -187,17 +187,23 @@ def prices(self) -> str | None:
             return 0
 
     @property
-    def certification(self) -> str | None:
+    def certification(self) -> bool | None:
```

Comment:
```suggestion
    def certification(self) -> bool:
```

Previously we returned None as the falsy value, now we return False.

(This will remove `nullable: true` from the OpenAPI spec.)